api/proto/relay/relay.proto

```diff
@@ -9,43 +9,23 @@ option go_package = "github.com/Layr-Labs/eigenda/api/grpc/relay";
 
 // Relay is a service that provides access to public relay functionality.
 service Relay {
-  // GetBlobs retrieves blobs stored by the relay.
-  rpc GetBlobs(GetBlobsRequest) returns (GetBlobsReply) {}
+  // GetBlob retrieves a blob stored by the relay.
+  rpc GetBlob(GetBlobRequest) returns (GetBlobReply) {}
 
   // GetChunks retrieves chunks from blobs stored by the relay.
   rpc GetChunks(GetChunksRequest) returns (GetChunksReply) {}
 }
 
 // A request to fetch one or more blobs.
-message GetBlobsRequest {
-  // The hashes of the blobs to retrieve. The order of these requested headers will
-  // be equal to the order of the blobs in the reply.
-  repeated bytes blob_hashes = 1;
-
-  // If this is an authenticated request, this should hold the ID of the requester. If this
-  // is an unauthenticated request, this field should be empty.
-  uint64 requester_id = 2;
-
-  // If this is an authenticated request, this field will hold a signature by the requester
-  // on the blob hashes being requested.
-  bytes requester_signature = 3;
+message GetBlobRequest {
+  // The key of the blob to fetch.
+  bytes blob_key = 1;
 }
 
 // The reply to a GetBlobs request.
-message GetBlobsReply {
-  // The blobs requested. The order of these blobs will be equal to the order of the
-  // requested header hashes.
-  repeated RequestedBlob blobs = 1;
-}
-
-// A blob that was requested or an error explaining why the requested blob could not be fetched.
-message RequestedBlob {
-  oneof data {
-    // The blob requested.
-    bytes blob = 1;
-    // An error (if any) that occurred while trying to fetch the blob.
-    RelayError error = 2;
-  }
+message GetBlobReply {
+  // The blob requested.
+  bytes blob = 1;
 }
 
 // Request chunks from blobs stored by this relay.
```

Comment:
I think there's an argument to be made that we only need to be able to address `(list of keys)\times(list of indices)` instead of `list of (keys,list of indices)`, since DA nodes will always be requesting the same set of indices for the blobs in a given batch. 

This will decrease the size of the request that the DA nodes are sending. I think this would have been a larger benefit for light node sampling since light nodes must send a list of indices. But maybe still worth considering.

Reply:
Change made.

Reply:
> since DA nodes will always be requesting the same set of indices for the blobs in a given batch

I don't think this is necessarily true. For any given blob, a node may have to download two different sets of index ranges because the assigned index range is different for each quorum?

Reply:
Good point. I suppose the the node could make one request per quorum / batch?